docs/conf.py

```diff
@@ -30,18 +30,30 @@
 extensions = [
     "numpydoc",
     "sphinx_gallery.gen_gallery",
+    # "qtgallery",
     "sphinx_automodapi.automodapi",
     "sphinx_automodapi.smart_resolver",
     "sphinx.ext.intersphinx",
 ]
 
 sphinx_gallery_conf = {
-    "filename_pattern": ".",
-    "image_scrapers": (qtgallery.qtscraper,),
-    "reset_modules": (qtgallery.reset_qapp,),
+    # Don't run any gallery examples, because they're not working on
+    # readthedocs at the moment
```

Comment:
😭   But it was so nice.

What was the problem? I assume it's not a problem of `xfvb` on rtd?

Reply:
I'm not sure, but `xvfb` just kept crashing. Might be due to some leaking/non-closed widgets, but the error message wasn't helpful enough to make a more educated guess.

Reply:
I guess we would need `sudo` access to an 18.04 VM without a window server to try it ourselves?